docker-compose.yaml

```diff
@@ -16,15 +16,18 @@ services:
           devices:
             - driver: nvidia
               count: all
-              capabilities: [gpu]
+              capabilities: [ gpu ]
   copilot_proxy:
-    image: moyix/copilot_proxy:latest
+    # For dockerhub version
+    # image: moyix/copilot_proxy:latest
+    # command: python3 -m flask run --host=0.0.0.0 --port=5000
     # For local build
-#    build:
-#      context: .
-#      dockerfile: copilot_proxy/Dockerfile
-    command: uvicorn app:app --host $API_HOST --port $API_PORT
+    build:
```

Comment:
Uncommenting build is only useful for local development, for production you should use images. That's why I commented out build by default.

Reply:
`moyix/copilot_proxy:latest` is ​​still not ready to use `uvicorn`, this change is to make the project usable instead of giving `command not found` errors. If the docker image gets updated after that, it should get updated here too.

Reply:
Ah yes, CI/CD for the base images should be setup next 😄. I think we should focus on that issue instead of making `build` the default. What do you think @moyix , what are your thoughts on building the images ?

Reply:
Hmm, I think using the Docker Hub images without building them is probably going be what 90% of people using this want. I did indeed forget to rebuild and push those after merging the PR though, so that's my fault. It would be great to have a GitHub Actions CI to build the containers and push them automatically, and that's probably preferable to make it build by default.